ops/pebble.py

```diff
@@ -2476,8 +2476,8 @@ def __init__(
         self._max_lookahead = max_lookahead
         self._max_boundary_length = max_boundary_length
 
-        self._buf = bytearray()
-        self._pos = 0  # current position in buf
+        self._buf: bytearray = bytearray()
+        self._pos: int = 0  # current position in buf
```

Comment:
Huh, it's odd that these types can't be inferred from the (obvious?) expressions on the right hand side... but I see that Pyright complains loudly without them.

Reply:
yeah .. even `int` 🤦🏽